File: src/main/java/org/evertones/controller/IndexController.java
package org.evertones.controller;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;

@Controller
public class IndexController {

    @RequestMapping(path = "/", method = RequestMethod.GET)
    public String get() {
        return "index";
    }

    @RequestMapping(path = "/index", method = RequestMethod.GET)
    public String getIndex() {
        return "index";
    }

    @RequestMapping(path = "/login", method = RequestMethod.GET)
    public String getLogin() {
        return "login";
    }

    @RequestMapping(path = "/construction", method = RequestMethod.GET)
    public String getConstruction() {
        return "underConstruction";
    }

}

File: src/main/scala/org/evertones/controller/IndexController.scala
package org.evertones.controller

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.{RequestMapping, RequestMethod}

@Controller
class IndexController {

    @RequestMapping(path = Array("/"), method = Array(RequestMethod.GET))
    def get(): String = {
        "index"
    }

    @RequestMapping(path = Array("/index"), method = Array(RequestMethod.GET))
    def getIndex: String = {
        "index"
    }

    @RequestMapping(path = Array("/login"), method = Array(RequestMethod.GET))
    def getLogin: String = {
        "login"
    }

    @RequestMapping(path = Array("/construction"), method = Array(RequestMethod.GET))
    def getConstruction: String = {
        "underConstruction"
    }

}
